scripts/release-2-commit.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+VERSION=$1
+: "${VERSION:?'The version string is needed as parameter 1.'}"
+
+set -e
```

Comment:
This exits the script on first error?

Reply:
Yep!  I added a brief comment about this in bd03b586 as well, for future us.